manifests/l/Logitech/GHUB/2023.6/Logitech.GHUB.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Logitech.GHUB
+PackageVersion: 2023.6

```

```suggestion
PackageVersion: 2023.6.430723
```
This should match control panel